src/components/projecteditor/control/backend.js

```diff
@@ -809,6 +918,14 @@ export default class Backend {
         else setTimeout(() => cb({ status: 1 }), 1);
     };
 
+    isDuplicateInode (projects, inode) {
```

Comment:
This should be moved into the `generateRandomInode` function, so we can be sure it always returns a unique inode.

Reply:
@Bashlund fixed in the latest commit